src/pybind/mgr/dashboard/frontend/e2e/rgw/users.e2e-spec.ts

```diff
@@ -2,6 +2,7 @@ import { Helper } from '../helper.po';
 
 describe('RGW users page', () => {
   let users;
```

Comment:
```suggestion
  let users: UsersPageHelper;
```

Reply:
The reason I don't set the users variable to UsersPageHelper is because I'm not importing it so it is marked as undefined:

![Screenshot from 2019-07-24 10-25-37](https://user-images.githubusercontent.com/47704447/61801905-933b9f00-adfd-11e9-8a8f-ed769d3dc3f2.png)

This has to do with the reformatting done in #29070 . I'm trying to use the Helper file as a sort of central registry so I can access any other pages functions without having to include the other pages po files sort of like what is outlined in #28592

Reply:
Of course if you set the type, yo have to import it.

This topic is out of scope so it was just a suggestion:
I see that there has been a cleanup of static typing, but I think that we are losing
the benefits of static typing (that is 1 of the strong points of "Type"script)
just to save a few import lines.

Reply:
This is now being addressed in #29324

Reply:
:+1: